services/movie.py

```diff
@@ -0,0 +1,31 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids=None, actors_ids=None):
+    queryset = Movie.objects.all()
+
```

Comment:
As I asked you in slack - here right now everything is ok, but we will ask students also to update this service - to add filter by title, if title is provided (`icontains`)